utils/js/cookies.js

```diff
@@ -0,0 +1,39 @@
+// Regex taken from https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie don't ask me how it works
+function getRegexForName (name){
+	return new RegExp(`(?:(?:^|.*;\\s*)${name}\\s*\=\\s*([^;]*).*$)|^.*$`);
+}
+
+function getOptions (options){
+	return Object.keys(options).map(key => {
+		let newKey = key;
+		if(key === 'maxAge'){
+			newKey = 'max-age';
+		}
+
+		if(options[key] instanceof Date){
+			options[key] = options[key] = options[key].toUTCString();
+		}
+
+		return `${newKey}=${options[key]}`;
+	});
+}
+
+function set (name, value, opts){
+	const options = getOptions(opts).join('; ');
+	const cookie = `${encodeURIComponent(name)}=${encodeURIComponent(value)}; ${options}`;
```

Comment:
Cos cookies are supposed to be url encoded according to the spec.  Browsers handle it either way in most cases, but just trying to be correct